static/schemas/2.0/skus.2.0.json

```diff
@@ -0,0 +1,248 @@
+{
+    "title": "Skus.2.0",
+    "definitions": {
+        "SkusRules": {
+            "description": "Possible Rules a Sku can use",
+            "type": "array",
+            "items": {
+                "type": "object",
```

Comment:
@benbp @yyscamper edited rules list to match those listed in documentation and specified types